opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -91,7 +93,7 @@ def _init_tracing(
 
 
 def _init_logging(
-    exporters: Dict[str, Sequence[LogExporter]],
+    exporters: Dict[str, Type[LogExporter]],
```

Comment:
It would be good to add tests to cover `_init_logging`

Reply:
Added, PTAL.

Reply:
Thanks! LGTM